tests/unit/test_miscutils.py

```diff
@@ -0,0 +1,61 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+
+import pytest
+
+from PyInstaller.utils.misc import load_py_data_struct, save_py_data_struct
+
+
+class TestLoadPyDataStruct(object):
```

Comment:
Can we stick to a test function rather than a test class seeing as it's already in its own file and doesn't need to be further grouped.

Reply:
I was going for consistency with `test_hookutils`, but you're right that it's unnecessarily complicated.